package.json

```diff
@@ -65,5 +68,15 @@
     ],
     "coverageDirectory": "../coverage",
     "testEnvironment": "node"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "{src,apps,libs,test}/**/*.ts": [
+      "eslint --fix"
+    ]
   }
```

Comment:
This might also be a little too opinionated but after automatic code formatting in the IDE, this is the best way to ensure that the code will be kept pristine.